docker-clean.sh

```diff
@@ -18,22 +19,22 @@
 
 export CURRENT_UID=$(id -u):$(id -g)
 
```

Comment:
as the root folder of the app is called data-collector, the docker container uses this name as container names by default. Thus, I had to change these names for the script to work out of the box.

Reply:
Container_Name works but: even though I delete the images, the image names are still the base folder name: data-collector. Thus, I added the -p options in the build and start script. I also had to replace the generic docker-compose stop with docker stop <container_names> for it to work on my machine.